pyefd.py

```diff
@@ -239,18 +240,55 @@ def plot_efd(coeffs, locus=(0.0, 0.0), image=None, contour=None, n=300):
     yt = np.ones((n,)) * locus[1]
 
     for n in _range(coeffs.shape[0]):
-        xt += (coeffs[n, 0] * np.cos(2 * (n + 1) * np.pi * t)) + (
-            coeffs[n, 1] * np.sin(2 * (n + 1) * np.pi * t)
-        )
-        yt += (coeffs[n, 2] * np.cos(2 * (n + 1) * np.pi * t)) + (
-            coeffs[n, 3] * np.sin(2 * (n + 1) * np.pi * t)
-        )
+        xt += coeffs[n, 0] * np.cos(2 * (n + 1) * np.pi * t) + coeffs[n, 1] * np.sin(2 * (n + 1) * np.pi * t)
+        yt += coeffs[n, 2] * np.cos(2 * (n + 1) * np.pi * t) + coeffs[n, 3] * np.sin(2 * (n + 1) * np.pi * t)
+
         ax = plt.subplot2grid((n_rows, N_half), (n // N_half, n % N_half))
         ax.set_title(str(n + 1))
         if contour is not None:
-            ax.plot(contour[:, 1], contour[:, 0], "c--", linewidth=2)
-        ax.plot(yt, xt, "r", linewidth=2)
+            ax.plot(contour[:, 0], contour[:, 1], "c--", linewidth=2)
+        ax.plot(xt, yt, "r", linewidth=2)
```

Comment:
This works well when it is plotted on top of an image (An `image` input is provided). Rewrite this to plot with `(xt, yt)` if no image is provided.

Reply:
Thanks for mentioning this usecase. I will adjust the code as suggested.